index.js

```diff
@@ -234,14 +236,15 @@ function SpotifyWebHelper(opts) {
 	};
 	this.player.seekTo = seconds => {
 		this.status.playing_position = seconds; // eslint-disable-line camelcase
+		this.player.emit('seek', seconds);
 		return this.player.play(this.status.track.track_resource.uri + '#' + parseTime(seconds));
 	};
 	this.status = null;
 	var seekingInterval = null;
 	var startSeekingInterval = function () {
 		seekingInterval = setInterval(() => {
-			this.status.playing_position += 0.25; // eslint-disable-line camelcase
-		}, 250);
+			this.status.playing_position += SEEK_INTERVAL_LENGTH / 1000; // eslint-disable-line camelcase
```

Comment:
Should we trigger a `seek` event here too? That way you can build a progress bar by just listening to the `seek` event. Otherwise the event just fires on manual changes.
(If so, also update the README text)